runtime/server_http_response.go

```diff
@@ -33,81 +32,61 @@ import (
 // ServerHTTPResponse struct manages request
 type ServerHTTPResponse struct {
 	responseWriter    http.ResponseWriter
-	Request           *ServerHTTPRequest
-	gateway           *Gateway
-	finishTime        time.Time
-	finished          bool
-	metrics           *EndpointMetrics
+	Req               *ServerHTTPRequest
```

Comment:
Renaming `Request` to `Req` is breaking change. It's not worth it.

Reply:
Trying to be consistent in the naming.  Why is this a breaking change?